taxi/tests/test_models.py

```diff
@@ -0,0 +1,49 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from taxi.models import Manufacturer, Car
+
+
+class ModelTests(TestCase):
+    def test_manufacturer_string(self):
+        manufacturer = Manufacturer.objects.create(
+            name="test_manufacturer",
+            country="test_country"
+        )
+        self.assertEqual(
+            str(manufacturer),
+            f"{manufacturer.name} {manufacturer.country}"
+        )
+
+    def test_driver_string(self):
+        driver = get_user_model().objects.create(
+            username="test_username",
+            password="test123",
+            first_name="test_first_name",
+            last_name="test_last_name",
+        )
```

Comment:
In the `test_driver_string` method, use `get_user_model().objects.create_user` instead of `create` to ensure that the password is hashed correctly. This is important for maintaining security and consistency with user creation.